test/Test/Data/List/Lazy.purs

```diff
@@ -270,6 +270,11 @@ testListLazy = do
   assert $ (take 2 (l [1, 2, 3])) == l [1, 2]
   assert $ (take 1 nil') == nil'
 
+  log "take should evaluate exactly n items which we needed"
+  assert let oops x = 0 : (oops x)
```

Comment:
sorry to nitpick here, but the parens around `oops x` aren't necessary and I think it would be better style to remove them, and also I think it would be a little nicer (and more in line with the style used elsewhere in this file) to use parens instead of `$` here too.

Reply:
So I see! parens are not unnecessary here. Thanks for your correction.